src/config-site/thembisa.llnl.gov.cmake

```diff
@@ -29,8 +29,8 @@ endif()
 VISIT_OPTION_DEFAULT(VISIT_THREAD OFF TYPE BOOL)
 VISIT_OPTION_DEFAULT(VISIT_PARADIS OFF TYPE BOOL)
 VISIT_OPTION_DEFAULT(VISIT_CREATE_XMLTOOLS_GEN_TARGETS OFF TYPE BOOL)
-VISIT_OPTION_DEFAULT(VISIT_BUILD_AVTEXAMPLES OFF TYPE BOOL)
-VISIT_OPTION_DEFAULT(VISIT_BUILD_DIAGNOSTICS OFF TYPE BOOL)
+VISIT_OPTION_DEFAULT(VISIT_ENABLE_EXAMPLES OFF TYPE BOOL)
+VISIT_OPTION_DEFAULT(VISIT_ENABLE_DIAGNOSTICS OFF TYPE BOOL)
```

Comment:
Just curios but what are `diagnostics`?

Reply:
The items in src/tools/dev/diagnostics: mpitest, networktest, exceptiontest, osmesatest.